doc/api/dns.md

```diff
@@ -560,10 +560,12 @@ be an array of objects with the following properties:
 added: v0.1.27
 -->
 
+<!--lint disable no-undefined-references list-item-bullet-indent-->
```

Comment:
This appears to be a small conflict with the v12 and v13. The `list-item-bullet-indent` can be removed once upgraded to v13